.github/workflows/update-contributors.yaml

```diff
@@ -0,0 +1,25 @@
+name: update-contributors
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 3 * * 5" # 03:00(UTC) every Friday 
+
+jobs:
+  updateContributors:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - run: |
+          make gen/contributions
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }} # `gh` command requires this token.
+      - name: Create Pull Request
```

Comment:
[ask] How does it work when nothing to date?

Reply:
thanks,
then no PR will be created and the action exits.


> If there are no changes (i.e. no diff exists with the checked-out base branch), no pull request will be created and the action exits silently.

https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#action-behaviour